classes/class-wc-connect-help-provider.php

```diff
@@ -2,20 +2,263 @@
 
 if ( ! class_exists( 'WC_Connect_Help_Provider' ) ) {
 
+	define( 'WOOCOMMERCE_CONNECT_MINIMUM_WOOCOMMERCE_VERSION', '2.6' );
+	define( 'WOOCOMMERCE_CONNECT_MINIMUM_JETPACK_VERSION', '4.0.2' );
+	define( 'WOOCOMMERCE_CONNECT_MAXIMUM_SCHEMA_AGE_SECONDS', 3 * DAY_IN_SECONDS );
```

Comment:
Yeah, that might be too generous. Maybe it should be 2 days since we have a daily cron. What do you think?